x12-edi/2.5/modules/ROOT/pages/index.adoc

```diff
@@ -60,25 +60,26 @@ For example, using the `005010` version and the 850 transaction set, the schema
 
 The message itself contains the following keys, some of which only apply to either the read operation or the write operation, as indicated:
 
+
 [%header%autowidth.spread]
 |===
 |Key name |Description
 |Delimiters |The delimiters used for the message. If used, this set is based on the last interchange processed by a read operation. If set for a write operation, this overrides the values in the module configuration. The characters in the string are interpreted based on position in the following order: data separator, sub-element separator, repetition separator, or `U` if none, segment terminator.
-|Errors (read only) |A list of errors that are associated with the input as a whole, or with interchanges with no valid transaction sets
-|FunctionalAcksGenerated (read only) |A list of `997` or `999` configured acknowledgments that are generated by the module during the read operation
+|Errors (read only) |A list of errors that are associated with the input as a whole, or with interchanges with no valid transaction sets.
+|FunctionalAcksGenerated (read only) |A list of `997` or `999` configured acknowledgments that are generated by the module during the read operation.
 |Group (write only) |Map of GS group header segment values used as defaults when writing groups
-|Interchange (write only) |Map of ISA interchange header segment values used as defaults when writing interchanges
-|InterchangeAcksGenerated (read only) |A list of TA1 acknowledgments that were generated by the module during the read operation
-|InterchangeAcksReceived (read only) |A list of TA1 acknowledgments that were received by the module during the read operation
-|InterchangeAcksToSend (write only) |A list of TA1 acknowledgments that are sent by the module during the write operation
+|Interchange (write only) |Map of ISA interchange header segment values used as defaults when writing interchanges.
+|InterchangeAcksGenerated (read only) |A list of TA1 acknowledgments that were generated by the module during the read operation.
+|InterchangeAcksReceived (read only) |A list of TA1 acknowledgments that were received by the module during the read operation.
+|InterchangeAcksToSend (write only) |A list of TA1 acknowledgments that are sent by the module during the write operation.
 |Transactions |A hierarchy of the transaction sets that are read by the module during a read operation, or are sent by the module during a write operation. The value of this top-level key is a map with standard versions as keys, in the form of `v005010`.
 For example, if you are using version `005010` `850` and `855` transaction sets, the Transactions would contain a map with one key, `v005010`. The value of this key would be another map, one with two keys, `850` and `855`. Each of these  contain a list of individual `850` and `855` transaction sets that are processed. If you're using `997` functional acknowledgments. Then `997` is the key for a list of 997 transaction sets.
```

Comment:
Again, this is a lot of text for a table column. Consider moving the example to a narrative discussion elsewhere, as appropriate.

Reply:
Copyedit:
```suggestion
For example, if you are using version `005010` with `850` and `855` transaction sets, the transactions contain a map with one key, `v005010`. The value of this key is another map, one with two keys, `850` and `855`. Each of these contain a list of individual `850` and `855` transaction sets that are processed. If you're using `997` functional acknowledgments, then `997` is the key for a list of 997 transaction sets.
```

Reply:
BREAK here to commit to prevent Github error and to enable writer to start work.